src/lfortran/parser/fixedform_tokenizer.cpp

```diff
@@ -889,6 +889,39 @@ struct FixedFormRecursiveDescent {
         return false;
     }
 
+    bool lex_common_block(unsigned char *&cur) {
+        push_token_advance(cur, "common");
+        int num_slash = 0;
+        while(!next_is_eol(cur)) {
+            // check if it is a "/"
+            if (*cur == '/') {
+                if (num_slash >= 2 ) {
+                    unsigned char *end = cur;
+                    // tokenize uptil here
+                    tokenize_until(end);
+                    if (*(cur-1) != ',') {
+                        YYSTYPE y2;
+                        Location loc;
+                        auto token = 271;
+                        tokens.push_back(token);
+                        std::string tk{","};
+                        y2.string.from_str(m_a, tk);
+                        stypes.push_back(y2);
+                        locations.push_back(loc);
```

Comment:
I think this is equivalent to:
```suggestion
                        push_token_no_advance(cur, ",");
```
Test it, but one of the `push_token_*` functions should work here.